addons/web/tests/test_read_progress_bar.py

```diff
@@ -76,16 +76,16 @@ def test_simple(self):
             {'x_country_id': c1.id, 'x_date': '2021-05-20', 'x_state': 'foo'},
             {'x_country_id': c1.id, 'x_date': '2021-05-21', 'x_state': 'foo'},
             {'x_country_id': c1.id, 'x_date': '2021-05-22', 'x_state': 'foo'},
-            {'x_country_id': c1.id, 'x_date': '2021-05-23', 'x_state': 'bar'},
             # week 22
+            {'x_country_id': c1.id, 'x_date': '2021-05-23', 'x_state': 'bar'},
```

Comment:
That's not correct. The test should be fixed to use standard (ISO) weeks to keep the current behaviour, then additional tests should be added to test variants: middle eastern (starts on saturday) and trad / american (starts on sunday).